backend/entities/Account.ts

```diff
@@ -23,6 +27,9 @@ export class Account {
     @Column()
     displayName: string;
 
+    @OneToMany(() => Leaderboard, (leaderboard) => leaderboard.account)
+    leaderboard: Leaderboard | undefined;
```

Comment:
Don't think it needs to be linked in both entities, should be enough to have ManyToOne in leaderboard entity.

Also there are some multiple empty lines and empty lines at the start of the file :)